src/utils/mtev_curl.c

```diff
@@ -125,6 +127,7 @@ mtev_curl_debug_function(CURL *handle, curl_infotype type, char *data, size_t si
 typedef struct {
   eventer_t e;
   curl_socket_t s;
+  uint16_t refcnt;
```

Comment:
please use ``stdatomic`` in new code

```suggestion
  atomic_uint_least16_t refcnt;
```